cla_public/templates/macros/subform.html

```diff
@@ -42,7 +42,9 @@
       <li>{{ _('If your nominated person misses the call you’ll need to complete this form again.') }}</li>
     </ul>
     {{ Form.group(form.thirdparty.full_name, field_attrs={'class': 'm-large'}) }}
-    {{ Form.group(form.thirdparty.relationship) }}
+    {{ Form.group(form.thirdparty.relationship,
+      '',
```

Comment:
What is this empty argument? I can see it in many but not all (eg `form.thirdparty.contact_number` directly below) instances

Reply:
Not 100% sure.  I just copied the pattern, including the empty argument.  I could see element classes were passed into it in some instances.  

Looking into if further, this passes extra classes into the group element, such as the conditional element for show-hide things.  

I am surprised to see it declared null, like you obviously are, but I just accepted it and carried on.

Reply:
I've just tried out removing it and it seems to work fine.  I can't see any difference in the rendered HTML.  I shall remove the remaining ones.